src/pages/LoadingPage.tsx

```diff
@@ -1,36 +1,31 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import FullScreenLoadingIndicator from '@components/FullscreenLoadingIndicator';
 import HeaderWithBackButton from '@components/HeaderWithBackButton';
 import ScreenWrapper from '@components/ScreenWrapper';
 import useThemeStyles from '@hooks/useThemeStyles';
 
-const propTypes = {
+type LoadingPageProps = {
     /** Method to trigger when pressing back button of the header */
-    onBackButtonPress: PropTypes.func,
-    title: PropTypes.string.isRequired,
-};
+    onBackButtonPress?: () => void;
 
-const defaultProps = {
-    onBackButtonPress: undefined,
+    /** Title of the Loading page */
```

Comment:
This comment feels unnecessary to me, can we get rid of it?

Reply:
Fine, I will fix it right away